### PR TITLE
Add Idyllic Pizza Pad

### PR DESCRIPTION
**Pull request by zykrah**
## Description

As per title.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/idyllic/pizzapad/keyboard.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        "cols": ["GP1", "GP0", "GP7"],
+        "rows": ["GP6", "GP29", "GP28"]
+    },
+    "usb": {
+        "device_version": "0.0.1",
+        "pid": "0x5050",
+        "vid": "0x7A79"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (zvecr) [inline keyboards/idyllic/pizzapad/keyboard.json:21]:
```suggestion
    "community_layouts": ["ortho_3x3"],
    "layouts": {
        "LAYOUT_ortho_3x3": {
```

File: keyboards/idyllic/pizzapad/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+		KC_P7,   KC_P8,   KC_P9,
+		KC_P4,   KC_P5,   KC_P6,
+		KC_P1,   KC_P2,   KC_P3
+	),
+
+	[1] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS
+	),
```
Comment (zvecr) [inline keyboards/idyllic/pizzapad/keymaps/default/keymap.c:18]:
```suggestion
	[0] = LAYOUT_ortho_3x3(
		KC_P7,   KC_P8,   KC_P9,
		KC_P4,   KC_P5,   KC_P6,
		KC_P1,   KC_P2,   KC_P3
	)
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/idyllic/pizzapad/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT_ortho_3x3(
+		KC_P7,   KC_P8,   KC_P9,
+		KC_P4,   KC_P5,   KC_P6,
+		KC_P1,   KC_P2,   KC_P3
+	)
+
```
Comment (waffle87) [inline keyboards/idyllic/pizzapad/keymaps/default/keymap.c:13]:
```suggestion
    [0] = LAYOUT_ortho_3x3(	
        KC_P7,   KC_P8,   KC_P9,
        KC_P4,   KC_P5,   KC_P6,
        KC_P1,   KC_P2,   KC_P3
    )
```

**Review by waffle87 — APPROVED** [review]
Otherwise LGTM.

**Review by drashna — APPROVED** [review]

**Pull request merged**